src/tests/account/test_eth_account.cairo

```diff
@@ -370,17 +369,17 @@ fn test_multicall() {
     let ret = account.__execute__(calls);
 
     // Assert that the transfers were successful
-    assert(erc20.balance_of(account.contract_address) == 200, 'Should have remainder');
-    assert(erc20.balance_of(recipient1) == 300, 'Should have transferred');
-    assert(erc20.balance_of(recipient2) == 500, 'Should have transferred');
+    assert_eq!(erc20.balance_of(account.contract_address), 200, "Should have remainder");
+    assert_eq!(erc20.balance_of(recipient1), 300, "Should have transferred in call 1");
+    assert_eq!(erc20.balance_of(recipient2), 500, "Should have transferred in call 2");
```

Comment:
in here we prefer an error string to disambiguate? should we include this in our styleguide #696?

Reply:
I think it is implicit in the third point [here](https://github.com/OpenZeppelin/cairo-contracts/issues/696#issuecomment-1878481216). Should we update the wording?